server/tests/integrations/Authentication.integration.test.ts

```diff
@@ -0,0 +1,67 @@
+import MemberServices from '../../src/services/MemberServices';
+import SessionServices from '../../src/services/SessionServices';
+
+import { dataSource, closeDatabase, startDatabase } from '../../src/db';
+import { INVALID_CREDENTIALS_ERROR_MESSAGE } from '../../src/utils/errorMessage';
+
+describe('Authentication integration test', () => {
+	beforeAll(async () => {
+		await startDatabase();
```

Comment:
For beter performance, should we set up a global jest file that launches databse for all tests instead of running the database for each test? What are your thoughts about it?

Reply:
The database start and close a connection in each file, not each test, to isolate the tests.